tests/compile-fail/panic/panic_abort3.rs

```diff
@@ -1,4 +1,6 @@
 // error-pattern: the program aborted execution
+// normalize-stderr-test: "\| +\^+" -> "| ^"
```

Comment:
should we start adding a scheme to add normalizations/comment flags for entire test folders?